packages/electron/app/package.json

```diff
@@ -8,7 +8,8 @@
   },
   "resolutions": {
     "electron": "3.1.0",
-    "debug": "^2.6.9"
+    "debug": "^2.6.9",
```

Comment:
We'll have to revert that change. It causes issues with macos builds

Reply:
Are you referring to `debug` and `xpc-connection` or just `debug`?